### PR TITLE
Update content on the service-maps page

### PR DESCRIPTION
**Pull request by bettymw**
# PR Checklist

- [ ] If you are proposing a new decision record document, used the right template for that
      - ([ADR](https://github.com/alphagov/forms/blob/main/ADR/ADRXXX-architecture-decision-record-template.md), [decision-record](https://github.com/alphagov/forms/blob/main/decision-record/DRXXX-decision-record-template.md), engagement, [research](https://github.com/alphagov/forms/blob/main/research/YYYY-MM-DD-template.md))
- [ ] Set yourself as the Assignee
- [ ] Tag anyone you would like to review, or @forms-design or @forms-devs
- [ ] Fill in the template below

---

# What

Describe what you have changed and why.

# How to review

Describe the steps required to test the changes.

For example:

1. Semantic: Do you agree with the changes?
1. Syntactic: Spelling, grammar, etc.

# Who can review

Describe who can review the changes. Or more importantly, list the people
that can't review, because they worked on it.

### CODE REVIEW

**Comment by bettymw** [comment]
I have created this doc to link different service maps. I have added a short description for each map and its last updated date. @hannahkc or @C-Harry if you get a chance, please have a look at the content on these to check its okay and feel free to suggest changes. appreciated much!

**Comment by hannahkc** [comment]
HI Betty - is there a trello card for this?

**Review by bettymw — COMMENTED** [review]
The changes look good to me and help with clarity

**Comment by bettymw** [comment]
@hannahkc there wasn't a Trello card as it was quite a small task to create the document and link the maps

**Comment by hannahkc** [comment]
Ah, that's because I have made a commit - so i can't review this. @C-Harry, @christophercameron-ixd or @hazalarpalikli could you approve this one? Feel free to give it a quick once over to check it looks ok too. :)

File: design/service-maps.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+This gives details and links to service maps we have created during the GOV.UK Forms design process.  
 
-What is a service map, sometimes called a service blueprint:
+A service map, sometimes called a service blueprint, is “a diagram that visualizes the relationships between different service components — people, props (physical or digital evidence), and processes — that are directly tied to touchpoints in a specific customer journey." [Source Nielsen Norman Group](https://www.nngroup.com/articles/service-blueprints-definition/)
 
-“A service blueprint is a diagram that visualizes the relationships between different service components — people, props (physical or digital evidence), and processes — that are directly tied to touchpoints in a specific customer journey." [Source Nielsen Norman Group](https://www.nngroup.com/articles/service-blueprints-definition/)
+GOV.UK Forms service maps visually represent users’ journeys and their interactions with the platform as they create forms. The maps have different levelw of detail and information depending on the specific purpose they’re created for and the product’s phase at the time of creation.
```
Comment (christophercameron-ixd) [inline design/service-maps.md:7]:
```suggestion
GOV.UK Forms service maps visually represent users’ journeys and their interactions with the platform as they create forms. The maps have different levels of detail and information depending on the specific purpose they’re created for and the product’s phase at the time of creation. 
```

**Review by christophercameron-ixd — APPROVED** [review]
One typo to fix then good to go

**Review by christophercameron-ixd — APPROVED** [review]

**Pull request merged**